src/units/unit.cpp

```diff
@@ -1786,6 +1786,19 @@ bool unit::resistance_filter_matches(const config& cfg, bool attacker, const std
 	return true;
 }
 
+static std::optional<int> ref_value(unit_ability_list resistance_abilities, const std::string& attribute)
```

Comment:
when it only supports max_value then it may be better to include max in function name

Reply:
@ProditorMagnus except what if a want introduce min_value later in 1.19, i prefer not to have change name anew, i propose to use "interval_value instead"

Reply:
static function removed and code reintroduced directly in function because an attribute only to check

Reply:
looks good